frontend/src/components/ProfileHistory/ProfileHistory.jsx

```diff
@@ -3,31 +3,61 @@ import { useState, useEffect, useContext } from "react";
 import axios from "axios";
 import { UserContext } from "../App/App";
 import { Link } from "react-router-dom";
+import ToggleButton from "react-bootstrap/ToggleButton";
+import ToggleButtonGroup from "react-bootstrap/ToggleButtonGroup";
+import Card from "react-bootstrap/Card";
 
 export default function ProfileHistory({ username }) {
     const [followsHistory, setFollowsHistory] = useState([]);
     const [likesHistory, setLikesHistory] = useState([]);
     const [dislikesHistory, setDislikesHistory] = useState([]);
-    const { user, setErrorMessage } = useContext(UserContext);
+    const { setErrorMessage } = useContext(UserContext);
+    const [mode, setMode] = useState([]);
 
     useEffect(() => {
         const fetchHistory = async () => {
             try {
                 const [followsResponse, likesResponse, dislikesResponse] =
                     await Promise.all([
-                        axios.get(`${import.meta.env.VITE_HOST}/follows/user/${username}`, { withCredentials: true, validateStatus: () => true }),
-                        axios.get(`${import.meta.env.VITE_HOST}/likes/user/${username}`, { withCredentials: true, validateStatus: () => true }),
-                        axios.get(`${import.meta.env.VITE_HOST}/dislikes/user/${username}`, { withCredentials: true, validateStatus: () => true })
+                        axios.get(
```

Comment:
did the linter do this?

Reply:
Or was it prettier? We need to make sure eslint and prettier do not contradict each other